modules/client-configuration/pages/manual-registration/salt-registration.adoc

```diff
@@ -110,3 +110,28 @@ You can adjust this manually to correct the error and register the cloned system
 
 
 For more information and instructions, see pass:c[xref:FILENAME.adoc#bp.chapt.suma3.troubleshooting.registering.cloned.salt.systems[]].
+
+
+=== SSL errors
+
+On {slsa}{nbsp}11 systems, salt clients can sometimes have SSL errors which make some operations unusable, including package management and bootstrapping.
+In this case, you will see an error like this:
+
+----
+Repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' is invalid.
+[|] Valid metadata not found at specified URL(s)
+Please check if the URIs defined for this repository are pointing to a valid repository.
+Skipping repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' because of the above error.
+Download (curl) error for 'www.example.com':
+Error code: Unrecognized error
+Error message: error:1409442E:SSL routines:SSL3_READ_BYTES:tlsv1 alert protocol version
+----
+
```

Comment:
I think it would be good to explain that apache in SUMA 4.0 is only allowing TLSv1.2 by default.
But SLES11 is so old that it does not support this protocol version.
To make SLES11 work with SUMA 4.0 the customer have to relax the protocol requirements.